tools/download-wheels.py

```diff
@@ -77,4 +79,8 @@ def download_wheels(version, wheelhouse):
     args = parser.parse_args()
 
     wheelhouse = os.path.expanduser(args.wheelhouse)
+    if not os.path.isdir(wheelhouse):
+        raise RuntimeError(f"{wheelhouse} wheelhouse directory is not present."
+                           " Perhaps you need to use the '-w' flag to specify one.")
```

Comment:
I used a new line for this in order to keep the line length < 80 characters.

Reply:
thanks @charris i have updated the pr. maybe this has been discussed before but it would be great to have basic pep8 check in the ci.